src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2419,6 +2419,13 @@ public ObjectId getHeadRev(String url, String branchSpec) throws GitException, I
     /** {@inheritDoc} */
     public Map<String, ObjectId> getRemoteReferences(String url, String pattern, boolean headsOnly, boolean tagsOnly)
             throws GitException, InterruptedException {
+        return getRemoteReferences(url, pattern != null ? Collections.singletonList(pattern) : Collections.EMPTY_LIST,
+                headsOnly, tagsOnly);
+    }
+
+    /** {@inheritDoc} */
```

Comment:
Look, I have my guidelines for the repositories where I am maintainer. Nicolas got push back from me for some changes he was proposing for one of "my" repositories and I am explaining **my** _preference_.

I prefer to require a concrete deliberate action on the behalf of a developer over an inaction.

I recognise that the tooling does "the mostly right thing". In fact no comment *should* be the same as `/** {@inheritDoc} */` which is fine, *but* adding the comment says "I have thought about this and spent my time typing this extra stuff to confirm my decision" while doing nothing does not convey any information.

Yes we could say "if you are not sure that the contract is the same then leave a `// TODO document` comment... That then requires a conscious effort to add... But if I forget to add it, we will never know that I didn't assess the documentation.

It is for this exact reason that **my preference in repositories that I can influence** is to use the doc comment to track methods where documentation has been considered. 

This repository is not one where I am a maintainer, so I am not dictating for this repository. I do believe my principle is logical and has merit, but each plugin maintainer is the ruler of their domain...

That Nicolas *and others* have trouble remembering which conventions hold sway where is the fault of plugin maintainers not defining a `CONTRIBUTING.md` file with their guidelines... So do not blame Nicolas for sticky bits of style, rather join the campaign for documenting style guidelines for plugins.